pkg/manifests/bundle.go

```diff
@@ -20,3 +20,17 @@ type Bundle struct {
 	V1CRDs         []*apiextensionsv1.CustomResourceDefinition
 	Dependencies   []*Dependency
 }
+
+func (b *Bundle) ObjsToValidate() []interface{} {
```

Comment:
I would prefer the full name here for func method `ObjectsToValidate` for clarity.